src/cli/sql.rs

```diff
@@ -271,6 +275,57 @@ fn process(
 		vec.push((stats, output));
 	}
 
+	tokio::spawn(async move {
+		let mut stream = match response.into_inner().stream::<Value>(()).await {
+			Ok(stream) => stream,
+			Err(error) => {
+				tx.send(Err(error.into())).await.ok();
+				return;
+			}
+		};
+		while let Some(Notification {
+			action,
+			data,
+			..
+		}) = stream.next().await
+		{
+			let message = match (json, pretty) {
+				// Don't prettify the SurrealQL response
+				(false, false) => {
+					let value = Value::from(map! {
+						String::from("action") => format!("{action:?}").into(),
+						String::from("data") => data,
+					});
+					value.to_string()
+				}
+				// Yes prettify the SurrealQL response
+				(false, true) => format!("-- Notification (action: {action:?})\n{data:#}"),
```

Comment:
Can we include the UUID here too? A user might be running multiple Live Queries

Reply:
I have added the live query ID and updated the description. I also changed the JSON representation to match the WebSocket protocol.